src/envoy/utils/mixer_control.cc

```diff
@@ -115,12 +113,12 @@ Grpc::AsyncClientFactoryPtr GrpcClientFactoryForCluster(
 bool ExtractInfoCompat(const std::string &nodeid, LocalNode *args) {
   auto parts = StringUtil::splitToken(nodeid, "~");
   if (parts.size() < 3) {
-    GOOGLE_LOG(ERROR) << "ExtractInfoCompat error len(nodeid.split(~))<3: "
-                      << nodeid;
+    GOOGLE_LOG(ERROR)
```

Comment:
Don't use GOOGLE_LOG,  use Envoy log, take a look at this code:
https://github.com/istio/proxy/blob/master/src/envoy/utils/config.cc#L54